CMakeLists.txt

```diff
@@ -3,12 +3,12 @@ cmake_minimum_required(VERSION 2.8.9)
 project(TCIABrowser)
 
 #-----------------------------------------------------------------------------
-set(EXTENSION_HOMEPAGE "https://www.slicer.org/wiki/Documentation/Nightly/Extensions/TCIABrowser")
+set(EXTENSION_HOMEPAGE "https://github.com/QIICR/TCIABrowser/blob/master/")
 set(EXTENSION_CATEGORY "Informatics")
 set(EXTENSION_CONTRIBUTORS "Alireza Mehrtash(SPL and BWH), Andrey Fedorov (SPL and BWH), Adam Li (GU), Justin Kirby (FNLCR)")
 set(EXTENSION_DESCRIPTION "A Module to connect to TCIA archive, browse the collections, patients and studies and download DICOM files to 3D Slicer.")
 set(EXTENSION_ICONURL "http://wiki.slicer.org/slicerWiki/images/9/92/TCIABrowser_logo.png")
-set(EXTENSION_SCREENSHOTURLS "http://www.slicer.org/slicerWiki/images/b/bf/TCIABrowser-screenshot.png")
+set(EXTENSION_SCREENSHOTURLS "https://github.com/SL2027/TCIABrowser/blob/master/TCIABrowser/Resources/Screenshot/Screenshot_1.png")
```

Comment:
Should this point to the main repo?  It's showing yours currently

Reply:
I fixed the URL in the latest commit below.